compiler/rustc_session/src/config.rs

```diff
@@ -1296,6 +1296,29 @@ impl CrateType {
             CrateType::Executable | CrateType::Cdylib | CrateType::Staticlib => false,
         }
     }
+
+    pub fn shorthand(&self) -> &'static str {
+        match *self {
+            CrateType::Executable => "bin",
+            CrateType::Dylib => "dylib",
+            CrateType::Rlib => "rlib",
+            CrateType::Staticlib => "lib",
```

Comment:
```suggestion
            CrateType::Staticlib => "staticlib",
```

Although `lib` also needs to be in the list.

Reply:
Good point...